ts/src/test/base/functions/test.number.ts

```diff
@@ -386,4 +386,5 @@ assert (Precise.stringLe ('3.1415', '3.14150000000000000000001'));
 // $this->assertSame (0,   Exchange::sum ());
 // $this->assertSame (2,   Exchange::sum (2));
 // $this->assertSame (432, Exchange::sum (2, 30, 400));
+// eslint-disable-next-line eol-last
```

Comment:
@olegtaranenko can we add a new line at the end of the file instead?

Reply:
I tried, but got error from Python.
try yourself, or we need to tune python lint or whatever